contexts/v1.1

```diff
@@ -0,0 +1,103 @@
+{
+  "@context": {
+    "@protected": true,
+    "id": "@id",
+    "type": "@type",
+
+    "alsoKnownAs": {
+      "@id": "https://w3id.org/security#alsoKnownAs",
```

Comment:
non blocking, I am struggling to remember where we landed on w3id ... since we have registries in this WG could we migrate what's on w3id to one of the w3c managed registries?

Reply:
Here's the latest discussion on w3id.org for the controller document: https://github.com/w3c/controller-document/issues/66#issuecomment-2311114405

Note that this URL https://w3id.org/security redirects to the security vocabulary maintained by the VCWG for the Data Integrity specifications. The only thing on w3id.org are redirect instructions to the VCWG document:

https://github.com/perma-id/w3id.org/blob/master/security/.htaccess

We need to add text to the following effect when we make the DID Core spec depend on the controller doc spec (effectively, "the hash of the base context is X... don't ever load it from the network"):

https://w3c.github.io/controller-document/#json-ld-context

Does that answer  your question @decentralgabe ? We've got a few more PRs we need to do to align the guidance in the controller document and DID Core (and hope to do those alignments in future PRs after we've based DID Core on the controller doc spec).

Reply:
@decentralgabe, the issue was raised (several times) and, at the moment, there is no change (in agreement with the w3c management on the DI CR).

Reply:
> there is no change (in agreement with the w3c management on the DI CR).

Meaning, W3C Management didn't see an issue with the way w3id.org is operated today and don't see a reason that W3C needs to take over management of the website. This is because we tell people to not load these contexts from the network, provide static copies, and provide hashes for the contexts in the specs.

Reply:
Thanks, yes that addresses my comment. The language you noted will be useful to add.